downstream/modules/platform/proc-controller-configure-analytics.adoc

```diff
@@ -4,6 +4,11 @@
 
 When you imported your license for the first time, you were automatically opted in for the collection of data that powers {Analytics}, a cloud service that is part of the {PlatformNameShort} subscription.
 
+.Prerequisites
+
+* A service account created.
+For more information see, link:https://docs.redhat.com/en/documentation/red_hat_hybrid_cloud_console/1-latest/html/creating_and_managing_service_accounts/proc-ciam-svc-acct-overview-creating-service-acct#proc-ciam-svc-acct-create-creating-service-acct[Creating a service account].
```

Comment:
we may want to mention that the service account used for Analytics needs to have Analytics Administrator viewer role in console.redhat.com